newa/cli.py

```diff
@@ -688,8 +688,10 @@ def cmd_schedule(ctx: CLIContext, arch: list[str]) -> None:
             architectures = Arch.architectures(
                 [Arch(a.strip()) for a in arch])
         else:
-            architectures = jira_job.erratum.archs if (
```

Comment:
Hi @happz , here I need your help. According to my testing, when loading `archs` from a YAML file then `jira_job.erratum.archs` is a list of strings while it is supposed to be a list of `Arch`. I believe the root cause is the missing converter for `archs` attribute which I am adding above but please review whether my assumption is correct. Functional-wise it seem to work.

Also, it seems that the same problem is with other attributes like
https://github.com/RedHatQE/newa/blob/main/newa/__init__.py#L798
which would explain why I need to access those as dictionary keys and not object attributes. Am I right?